scores.py

```diff
@@ -3,38 +3,45 @@
 
 # TODO: Restructure the JSON to be more 'normal'.
 
+import os
+import json
+import logging
+logging.basicConfig(level=logging.INFO)
+
 def score_individual_by_age(individual):
     """Score an individuals 'COVID risk' by age.
 
     See the analysis within stats/age_score.r to understand where the
     scores come from. Scores are 'odds ratios' of COVID-19 CFR risk.
 
-    TODO: Read this data from file?
-
     """
     if individual.age is None:
         return None
 
-    if individual.age < 9:
-        return 0
+    age_risk_scores = {}
+
+    age_risk_scores_fn = 'data/age_risk_scores.json'
+    if os.path.exists(age_risk_scores_fn):
+        with open(age_risk_scores_fn) as fh:
+            staging = json.load(fh)
+            prev_key = None
+            for key, val in staging.items():
+                int_key = int(key)
+                age_risk_scores[int_key] = float(val)
+                if prev_key is not None and int_key <= prev_key:
+                    logging.warning(f"{age_risk_scores_fn} is incorrectly ordered. {int_key} before {prev_key}")
```

Comment:
Dude that's ugly ;-)

Why not just read the data as is, and then sort by age later?

I have a feeling that there is a better way to do it overall, but that's the first simpler solution that comes to mind.

Reply:
>why not ...
1. JSON does not ints as dict keys. Python does allows ints as dict keys, and that fits more closely to the behavior of the original code.
2. Sorting later is of course possible, However if the data isn't already in sorted order the odds of a human making an error when populating the data goes way up. This seemed like it would enforce good user behavior, and warn if it wasn't being observed, even when the code was able to find a computational solution.